checkov/kubernetes/runner.py

```diff
@@ -251,7 +252,19 @@ def mutate_kubernetes_graph_results(
                 entity_file_path = entity[CustomAttributes.FILE_PATH]
                 entity_file_abs_path = _get_entity_abs_path(root_folder, entity_file_path)
                 entity_id = entity[CustomAttributes.ID]
-                entity_context = self.context[entity_file_path][entity_id]
+                # Deal with nested pods within a deployment.
```

Comment:
I would separate this block to a function with a relevant name, which will simplify the readability of the code.